backend/backend.methods.js

```diff
@@ -121,9 +121,26 @@ function initializeCollections(config) {
   })
 }
 
+function dropDatabase(config, cb) {
+  const { dbUrl, dbName } = config
+  MongoClient.connect(dbUrl, {useNewUrlParser: true}, async (err, client) => {
+    try {
+      const db = client.db(dbName)
+      await db.collection('drivers').drop();
+      await db.collection('routes').drop();
```

Comment:
testowałeś to? Bo po dropie musisz jeszcze założyć kolekcje na nowo, inaczej wszystko się rozleci.